src/main/resources/sql/create.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE IF NOT EXISTS "buyer" (
+  "buyerId" SERIAL PRIMARY KEY,
+  "buyerName" TEXT NOT NULL,
+  "buyerSurname" TEXT NOT NULL,
+  "age" SMALLINT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS "shop" (
+  "shopId" SERIAL PRIMARY KEY,
+  "shopName" TEXT NOT NULL UNIQUE,
+  "address" TEXT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS "plu" (
```

Comment:
I agree with @kortov. This name is not clear . What is plu ?

Reply:
Ok, i will do it.